libphobos/m4/druntime/libraries.m4

```diff
@@ -3,6 +3,30 @@
 #
 
 
+# DRUNTIME_LIBRARIES_DLOPEN
+# -----------------------
+# Check whether -ldl is required for dlopen.
+AC_DEFUN([DRUNTIME_LIBRARIES_DLOPEN],
+[
+AC_SEARCH_LIBS([dlopen], [dl])
+])
+
+
+# DRUNTIME_LIBRARIES_PLATFORM
+# -------------------------
+# Check for platform-specific library dependencies.
+AC_DEFUN([DRUNTIME_LIBRARIES_PLATFORM],
+[
+case ${target_os} in
+  mingw32*)
+      AC_SEARCH_LIBS([gethostbyname], [ws2_32])
+    ;;
```

Comment:
This is a library that you always want to link to, correct?  As opposed to others where you *may* need to link against it.